src/workerd/api/worker-rpc.h

```diff
@@ -28,30 +28,30 @@ namespace workerd::api {
 // separate calls.
 constexpr size_t MAX_JS_RPC_MESSAGE_SIZE = 1u << 20;
 
-// A WorkerRpc object forwards JS method calls to the remote Worker/Durable Object over RPC.
-// Since methods are not known until runtime, WorkerRpc doesn't define any JS methods.
+// A JsRpcCapability object forwards JS method calls to the remote Worker/Durable Object over RPC.
+// Since methods are not known until runtime, JsRpcCapability doesn't define any JS methods.
 // Instead, we use JSG_WILDCARD_PROPERTY to intercept property accesses of names that are not known
 // at compile time.
 //
-// WorkerRpc only supports method calls. You cannot, for instance, access a property of a
+// JsRpcCapability only supports method calls. You cannot, for instance, access a property of a
 // Durable Object over RPC.
 //
-// The `WorkerRpc` type is used to represent capabilities passed across some previous JS RPC call.
-// It is NOT the type of a Durable Object stub nor a service binding. Those are instances of
+// The `JsRpcCapability` type is used to represent capabilities passed across some previous JS RPC
+// call. It is NOT the type of a Durable Object stub nor a service binding. Those are instances of
 // `Fetcher`, which has a `getRpcMethod()` call of its own that mostly delegates to
-// `WorkerRpc::sendWorkerRpc()`.
-class WorkerRpc: public jsg::Object {
+// `JsRpcCapability::sendRpc()`.
+class JsRpcCapability: public jsg::Object {
 public:
-  WorkerRpc(IoOwn<rpc::JsRpcTarget::Client> capnpClient)
+  JsRpcCapability(IoOwn<rpc::JsRpcTarget::Client> capnpClient)
       : capnpClient(kj::mv(capnpClient)) {}
 
   // Serializes the method name and arguments, calls customEvent to get the capability, and uses
   // the capability to send our request to the remote Worker. This resolves once the RPC promise
   // resolves.
   //
-  // This is a static helper, no `WorkerRpc` object is needed. This is the shared implementation
-  // between `WorkerRpc::getRpcMethod()` and `Fetcher::getRpcMethod()`.
-  static jsg::Promise<jsg::Value> sendWorkerRpc(
+  // This is a static helper, no `JsRpcCapability` object is needed. This is the shared
+  // implementation between `JsRpcCapability::getRpcMethod()` and `Fetcher::getRpcMethod()`.
+  static jsg::Promise<jsg::Value> sendRpc(
```

Comment:
nit: This is probably fine, but I hesitated for a moment when I saw this method rename. It makes sense of course, but now we have `JsRpcCapability::sendRpc()` _and_ `JsRpcSessionCustomEventImpl::sendRpc()`. For anyone who looks at this code for the first time and isn't already familiar with how we dispatch `CustomEvent`s, this could be pretty confusing (a quick search for `sendRpc()` brings up a ton of results.

Maybe it's not an issue given we've got clangd and can find references, but figured I might as well mention it since overloading of names has been definitely been a pain point in the past. Maybe `sendJsRpc()`?

Reply:
Yeah that's a good call.